src/go/k8s/tests/e2e/scale-up/02-two-replicas.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
```

Comment:
Any reason why the cluster spec in 01 includes various fields like configuration and resources but 02 doesn't? Perhaps neither need it (or both)?

Reply:
so the first one applies the resource for the first time, so has to specify everything, the other step just updates the existing resource, so it can only contain subset. I can give you a demo of kuttl next week if you want to understand how all that goes :)

Reply:
Ah, got it. Thanks!